pages/guides/webhooks/validating-webhooks.mdx

```diff
@@ -62,37 +62,46 @@ not available
 ```
 
 ```javascript
-var verifyGatewaySignature = function (
+var crypto = require("crypto");
+const fromBuffer = require("buffer").Buffer.from;
```

Comment:
Maybe we make this `var` instead of `const` (for now) just to be consistent?

Reply:
Also, let's rename this to `bufferFrom` (my bad)

Reply:
ok